frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -1202,6 +1196,23 @@ function ReaderBookmark:showBookmarkDetails(item)
     return true
 end
 
+-- Refresh the bookmark list whenever details may have been edited
+function ReaderBookmark:refreshBookmarkDetails()
```

Comment:
I'd call it `_updateBookmarkList` and set a local function withinin `showBookmarkDetails()` as it is not used outside.